apps/modernization-ui/src/pages/patient/profile/identification/IdentificationsTable.tsx

```diff
@@ -75,13 +78,44 @@ export const IdentificationsTable = ({ patient }: Props) => {
     const { changed } = useProfileContext();
 
     const [isActions, setIsActions] = useState<any>(null);
-    const [identifications, setIdentifications] = useState<PatientIdentification[]>([]);
+
+    const asTableBody =
```

Comment:
Should this be moved into a separate component?

Reply:
This is only a function that returns an object for the tableBody so I don't think it should be moved into a separate component, any thoughts?

Reply:
Ok, thats fine. Just checking if are missing an opportunity to dry up  our code.